content/guides/docker-concepts/running-containers/overriding-container-defaults.md

```diff
@@ -0,0 +1,194 @@
+
+    ```console
+    $ docker run -d -e POSTGRES_PASSWORD=secret -p 5433:5432 postgres
+    ```
+
+    This will start a new Postgres container in the background, listening on the standard container port `5432` but mapped to port `5433` on the host machine. You override the host port just to ensure that this new container doesn't conflict with the existing running container.
+
+3. Verify if both containers are running via the Docker Dashboard.
+
+    ![A screenshot of Docker Dashboard showing the running instances of Postgres containers](images/running-postgres-containers.webp?border=true)
+
+### Run Postgres container in a controlled network
+
+By default, containers automatically connect to a special network called a bridge network when you run them. This bridge network acts like a virtual bridge, allowing containers on the same host to communicate with each other while keeping them isolated from the outside world and other hosts. It's a convenient starting point for most container interactions. However, for specific scenarios, you might want more control over the network configuration.
+
+Here's where the `--network` flag with the `docker run` command comes in.
+
+Follow the steps to see how to connect a Postgres container to a custom network.
+
+1. Create a network by using the following command:
+
+    ```console
+    $ docker network create mynetwork
+    ```
+
+2. Verify the network by running the following command:
+
+    ```console
+    $ docker network ls
+    ```
+
+3. Connect Postgres to the existing network by using the following command:
+
+    ```console
+    $ docker run -d -e POSTGRES_PASSWORD=secret -p 5434:5432 --network mynetwork postgres
+    ```
+
+    This will start Postgres container in the background, mapped to the host port 5434 and attached to the `mynetwork` network. You passed the `--network` parameter to override the container default by connecting the container to custom Docker network for better isolation and communication with other containers.
+
+    > **Inspecting the container network**
+    >
+    > You can use `docker network inspect` command to see if the container is tied to this new bridge network.
+    { .tip }
+
+### Manage the resources
+
+By default, containers are not limited in their resource usage. However, on shared systems, it's crucial to manage resources effectively. It's important not to let a running container consume too much of the host machine's memory.
+
+This is where the `docker run` command shines again. It offers flags like `--memory` and `--cpus` to restrict how much CPU and memory a container can use.
+
+```console
+$ docker run -d -e POSTGRES_PASSWORD=secret --memory="512m" --cpus=".5" postgres
+```
+
+The `--cpus` flag specifies the CPU quota for the container. Here, it's set to half a CPU core (0.5) whereas the `--memory` flag specifies the memory limit for the container. In this case, it's set to 512 MB.
+
+### Override the default `CMD` and `ENTRYPOINT` in Docker Compose
```

Comment:
I think this section should not only show the compose example, but also overriding default commands for `docker run`.

Reply:
I added a section for overriding the container default with`docker run`. Do check and let me know if it looks good.